index.html

```diff
@@ -0,0 +1,83 @@
+<!DOCTYPE html>
+<html lang="en">
+<main id="main">
+  <head> 
+    <meta charset="UTF-8">
+    <title >FCC: Tribute page</title>
+   
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <h1 id = "title"> Dr. Norman Borlaug </h1>
+    <p id="title">
+      The man who saved a billion lives
+    </p>
+    <div id="img-div">
+      <img  id="image" src="tribute-page-main-image.jpg" alt="image of biologist">
+      <p id="img-caption">
+        Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.
+      </p>
+    </div><br>
+
```

Comment:
It is a good practice to set up linters for your work. This will make your code cleaner and easier to follow and eliminate any unuseful extra information.

Reply:
- [ ] Always make sure the line or block under which the comments are written is the one that needs to be addressed. Here the comment is about linters set up but the block selected is just some piece from `index.html` file. Kindly take this comment to the concerned file or if there is no file to relate with you can keep this comment on the review summary field. 
- [ ] OPTIONAL: Here, you can also provide link to the general requirements.